tools/install_builder_prerequisites.sh

```diff
@@ -130,11 +132,16 @@ function install_packages {
       'libclang-dev'
       'libsqlite3-dev'
     )
-    sudo apt-get update
+    if [[ "${OMIT_SUDO}" == "false" ]]; then
+      maybe_sudo="sudo"
```

Comment:
nit, feel free to ignore: we could as well set the global `SUDO="sudo"` at L34 and have L41 set `SUDO=""` and then just use `$SUDO` at L142/L144.  It would just avoid an extra boolean global and the logic at L135-139 (which is a little circuitous with a double negative -- "if we're not omitting sudo, then set maybe_sudo to include it")